flink-core/src/main/java/org/apache/flink/configuration/HistoryServerOptions.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.flink.configuration;
+
+import org.apache.flink.annotation.PublicEvolving;
+
+import static org.apache.flink.configuration.ConfigOptions.key;
+
+/**
+ * The set of configuration options relating to the HistoryServer.
+ */
+@PublicEvolving
+public class HistoryServerOptions {
+
+	/**
+	 * The interval at which the HistoryServer polls {@link HistoryServerOptions#HISTORY_SERVER_DIRS} for new archives.
+	 */
+	public static final ConfigOption<Long> HISTORY_SERVER_REFRESH_INTERVAL =
+		key("historyserver.archive.fs.refresh-interval")
+			.defaultValue(10000L);
+
+	/**
+	 * Comma-separated list of directories which the HistoryServer polls for new archives.
+	 */
+	public static final ConfigOption<String> HISTORY_SERVER_DIRS =
+		key("historyserver.archive.fs.dirs")
```

Comment:
I would keep the default as dirs to make it clear that we accept multiple directories. To avoid the misconfiguration we could add a deprecated key "historyserver.archive.fs.dir".